README.rst

```diff
@@ -1,4 +1,16 @@
 cycler: composable cycles
 =========================
 
+.. image:: https://img.shields.io/pypi/v/cycler.svg?style=flat
+    :alt: PyPI Version
+    :target: https://pypi.python.org/pypi/cycler
+
+.. image:: https://img.shields.io/pypi/pyversions/cycler.svg
```

Comment:
Hm, there is no 3.6, it looks like `setup.py` needs an update.

Reply:
Will be updated when https://github.com/matplotlib/cycler/pull/43 is merged and released to PyPI.

Reply:
do we have to re-release to pypi to add support for a new version of python?  That is a bit annoying...

Reply:
The version on this badge comes from PyPI, and you don't have to release to PyPI but it is the most common place for Python libraries.

There have been some changes since the last 0.10.0 nearly two years ago: https://github.com/matplotlib/cycler/compare/v0.10.0...master

I can remove this badge if you prefer.